packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -248,6 +247,7 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
           );
 
           component = missingComponentComponent ?? MissingComponent;
+          isEmpty = true;
```

Comment:
Do we need to care about particular cases when we have _missingComponent_, _hiddenRendering_ and _empty_ component?
Probably we can just wrap element by _ErrorBoundary_ in all the cases.
At least for me still makes sense to wrap _hiddenRendering_, _missingComponent_ by ErrorBoundary since there is a way to provide custom component, so still something can fail

Reply:
I'm not sure if components that indicate of errors should be also be wrapped in error boundary... partially because it's an extra overhead for components that should just show up fast, partially because error boundary will output misleading info. I.e. in case of an error in custom missingComponent implementation: error boundary will show the name of the component that is missing when error happens in missingComponent.
What do you think?